test/validation/api/pktio/pktio.c

```diff
@@ -3316,7 +3316,6 @@ static void pktio_test_pktout_compl(bool use_plain_queue)
 		opt.queue = compl_queue[i];
 		opt.mode = ODP_PACKET_TX_COMPL_ALL;
 		odp_packet_tx_compl_request(pkt_tbl[i], &opt);
-
```

Comment:
This patch (linux-gen: packet: implement tx completion event APIs) still this (accidental) modification of test/validation/api/pktio/pktio.c file.